pages/people.tsx

```diff
@@ -1,13 +1,13 @@
 import type { NextPage } from "next";
 
 const People: NextPage = () => {
-  return (
-    <>
-      <div className="bg-main-bg  text-white min-h-screen min-w-full">
-        <h1 className="">People</h1>
-      </div>
-    </>
-  );
+	return (
+		<>
+			<div className="bg-main-bg  text-white min-h-screen min-w-full">
+				<h1 className="">People</h1>
```

Comment:
Is your prettier at default settings. if not do that, otherwise we will get unwanted changes in every commit